doc/conf.py

```diff
@@ -22,9 +20,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = setup_args['version']
+version = "1.7"
 # The full version, including alpha/beta/rc tags.
-release = setup_args['version']
+release = "1.7dev7"
```

Comment:
Odd that it works on Python 2 but not Python 3. Seems we will have to find a way to get it working on both as having another place to set the version string is just going to be a pain unless it is done from setup.py.